src/test/send_images.cpp

```diff
@@ -66,14 +67,16 @@ int main( int argc, char **argv )
     msg.step = width * 3;
 
     int count = 0;
+    std::default_random_engine seed;
     while( ros::ok() )
     {
       for( int x = 0; x < width; x++ )
       {
         for( int y = 0; y < height; y++ )
         {
           int index = (x + y * width) * 3;
-          long int rand = random();
+          std::uniform_int_distribution<int> random(0, RAND_MAX);
```

Comment:
This one maps pseudo random numbers into a random numbers of a specific, here uniform, distribution.
```suggestion
          std::uniform_int_distribution<int> uniform(0, RAND_MAX);
```